waterlog/src/Components/FaultySensor.js

```diff
@@ -94,10 +97,12 @@ class FaultySensor extends Component {
 					<small>(surrounding sensors have 100% waterflow)</small>
 				</div>
 				<img
-					id="map-toggle"
-					src={this.state.mapExpanded === false ? 'images/map_expand.png' : 'images/map_close.png'}
-					alt="segment-map"
-					onClick={() => this.handleMapExpand()}
+				 id="map-toggle"
+				 src={this.state.mapExpanded === false ?
+					 map_expand :
+					 map_close}
+				 alt="segment-map"
```

Comment:
This does not have to be dash-notation its just plain text (In which case this is fine just letting you know in case you were unaware

Reply:
noted